doc/source/user_guide/tutorial_parallelization_dask.ipynb

```diff
@@ -0,0 +1,364 @@
+    "Currently, we don't provide any dataset that is large enough to show the true advantages of dask.\n",
+    "To overcome this limitation, we will use a random number generator to create the data. We encourage you to adapt this function to load your dataset from disk."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "def load_my_data(index, shape=(2048, 2048), max_index=255):\n",
+    "    \"\"\"A simple loading function.\n",
+    "\n",
+    "    Return an image that is made of guassian noise centered about\n",
+    "    `index/max_index` with standard deviation equal to 20/max_index\n",
+    "    \"\"\"\n",
+    "    # Make sure to use independend random number generators otherwise\n",
+    "    # parallel code might have conflicts\n",
+    "    r = np.random.RandomState(index)\n",
+    "    image =  r.normal(loc=((index+1) % max_index)/max_index, scale=20/max_index, size=shape)\n",
+    "    return np.clip(image, 0, 1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "images = []\n",
+    "for i in range(0, N_images):\n",
+    "    image = load_my_data(i, shape=shape)\n",
+    "    images.append(image)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "input_variances = []\n",
+    "output_variances = []\n",
+    "output_images = []\n",
+    "\n",
+    "for image in images:\n",
+    "    input_variance = np.var(image)\n",
+    "    # Now we compute the output image as a processed version off the input\n",
+    "    # image\n",
+    "    output_image = filters.gaussian(image, 10)\n",
+    "    output_variance = np.var(output_image)\n",
+    "    output_images.append(output_image)\n",
+    "\n",
+    "    # Store all the results\n",
+    "    input_variances.append(input_variance)\n",
+    "    output_variances.append(output_variance)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "# Save\n",
+    "for i, image in enumerate(output_images):\n",
+    "    with warnings.catch_warnings():\n",
+    "        warnings.filterwarnings('ignore', 'Possible precision loss')\n",
+    "        image_ubyte = img_as_ubyte(image)\n",
+    "    filename = os.path.join(save_directory,\n",
+    "                            'image_{i:2d}.bmp'.format(i=i))\n",
+    "    imageio.imwrite(filename, image_ubyte)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### A note on memory usage\n",
+    "\n",
+    "We find that in many cases this kind of organization very useful during prototyping stages:\n",
+    "1. All images can be easily accessed from the variables `images`\n",
```

Comment:
The list is not correctly rendered.

Reply:
![image](https://user-images.githubusercontent.com/90008/56257237-96eeb580-6099-11e9-9e3e-dcf6b8a78278.png)

seems to be OK for me

Reply:
I added some indentation